example-github/src/main/java/feign/example/github/GitHubExample.java

```diff
@@ -53,4 +64,51 @@ public static void main(String... args) throws InterruptedException {
     String login;
     int contributions;
   }
+
+  static class ClientError {
```

Comment:
would be neat if this could extend RuntimeException, that would save some boilerplate in the example.. is that possible to make without gson getting complicated?